sdk/test-utils/recorder/karma.conf.js

```diff
@@ -4,7 +4,6 @@ process.env.CHROME_BIN = require("puppeteer").executablePath();
 require("dotenv").config({ path: "../.env" });
 
 process.env.RECORDINGS_RELATIVE_PATH = relativeRecordingsPath();
-process.env.RECORDING_ASSETS_PATH = relativeAssetsPath();
```

Comment:
changelog

Reply:
Looks like we forgot to add one for the asset sync integration in the first place. I just added a generic message to encompass both.